packages/wrangler/templates/middleware/middleware-miniflare3-json-error.ts

```diff
@@ -30,3 +30,4 @@ const jsonError: Middleware = async (request, env, _ctx, middlewareCtx) => {
 };
 
 export default jsonError;
+export const wrap = undefined;
```

Comment:
What is this?

Reply:
With this esbuild bump it now gives a warning like this
```
Import "wrap" will always be undefined because there is no matching export in "../../wrangler2/packages/wrangler/templates/middleware/middleware-miniflare3-json-error.ts" [import-is-undefined]    ../../../../../tmp/tmp-7210-764jhlLoLddp/middleware-insertion-facade.js:3:42:     
 const envWrappers = [__MIDDLEWARE_0__.wrap].filter(Boolean);
```
Instead of suppressing for this case I just fixed the warning